### PR TITLE
remove calls to super.lifecycle from existing hooks

### PR DESCRIPTION
**Pull request by felangel**
- Addresses #75

### CODE REVIEW

**Comment by rrousselGit** [comment]
Unless you want to do the change discussed on #75 yourself, we can close this.

**Comment by felangel** [comment]
Just took a crack at it. Let me know if I missed something 👍

File: lib/src/framework.dart [hunk 1]
```diff
@@ -71,7 +71,6 @@ bool debugHotReloadHooksEnabled = true;
 ///
 ///   @override
 ///   void initState() {
-///     super.initState();
```
Comment (rrousselGit) [inline lib/src/framework.dart]: That one and the `super.dispose` should stay

Comment (felangel) [reply lib/src/framework.dart]: oops my fault!

File: README.md [hunk 2]
```diff
@@ -29,21 +29,18 @@ class _ExampleState extends State<Example> with SingleTickerProviderStateMixin {
 
   @override
   void initState() {
-    super.initState();
```
Comment (rrousselGit) [inline README.md]: We need that one too! 😅

**Comment by rrousselGit** [comment]
LGTM after the last comment and a run of `flutter format`

**Comment by felangel** [comment]
Done 👍 (that's what I get for doing a "replace-all" at 1am 😝)

**Comment by rrousselGit** [comment]
Looks good, thank a lot!

**Pull request merged**